src/github.com/getlantern/protected/protected.go

```diff
@@ -127,6 +132,11 @@ func Resolve(addr string) (*net.TCPAddr, error) {
 //   specified system device (this is primarily
 //   used for Android VpnService routing functionality)
 func Dial(network, addr string, timeout time.Duration) (net.Conn, error) {
+	protect, _ := getCurrent()
+	if protect == nil {
```

Comment:
This allows us to use protected.Dial even when we're not on mobile, by simply falling back to net.DialTimeout